frontend/src/components/AnswerList.js

```diff
@@ -0,0 +1,25 @@
+import styled from "styled-components/macro";
+
+export default function AnswerList({possibleAnswers}) {
+    return (
+        <Wrapper>
+            {possibleAnswers.map(answer => (
+                <li key={possibleAnswers.indexOf(answer)}>{answer}</li>
+            ))}
```

Comment:
This is the same as:
```suggestion
            {possibleAnswers.map((answer,index) => (
                <li key={index}>{answer}</li>
            ))}
```
which is not so great, see here for reference: https://robinpokorny.medium.com/index-as-a-key-is-an-anti-pattern-e0349aece318

Maybe you can find something else that is unique and  can be used as a key?